apps/firestorm_web/test/unit/earmark/auto_linker_test.exs

```diff
@@ -7,4 +7,10 @@ defmodule FirestormWeb.Earmark.AutoLinkerTest do
     expected = "<a href=\"http://slashdot.org\">http://slashdot.org</a><br/>"
     assert AutoLinker.run(body) == expected
   end
+
+  test "does nothing to a naked URL wrapped in quotes" do
+    body = "\"http://slashdot.org\""
```

Comment:
I agree. Just trying to keep moving forward. We can make better test cases as we go, then rewrite something nice to satisfy them eventually. Ideally I'd dig in when Earmark has done its parsing and examine stuff there - just don't know yet. For now, I think it gets us moving though :)